src/test/java/plugins/DashboardViewPluginTest.java

```diff
@@ -28,4 +62,637 @@ public void configure_dashboard() {
         j.getLastBuild().shouldSucceed();
 
     }
+
+    @Test
+    public void jobsGridPortlet_fillColumnsFirst() throws MalformedURLException {
```

Comment:
It would be simpler if MalformedURLException would be converted to a RuntimeException in getJob method so that no throws is required.